synapse/cortex.py

```diff
@@ -1252,6 +1279,192 @@ async def initServiceStorage(self):
 
         await self.auth.addAuthGate('cortex', 'cortex')
 
+        await self._bumpCellVers('cortex:storage', (
+            (1, self._storUpdateMacros),
+        ), nexs=False)
+
+    async def _storUpdateMacros(self):
+        for name, node in await self.hive.open(('cortex', 'storm', 'macros')):
+
+            try:
+
+                info = node.valu
```

Comment:
Can we make a new dictionary containing exactly what we need from `node.valu` instead of operating on the in-memory dictionary held by the hive?